src/lib/BUILD.bazel

```diff
@@ -0,0 +1,46 @@
+load("//tools:sass_bundle.bzl", "sass_bundle")
+
+
+sass_bundle(
+  name = "theming_bundle",
+  srcs = [
```

Comment:
introduce a constant in the top-level BUILD file?

`ALL_LIBS=["autocomplete", "badge]`
then here it's just
`srcs = ["//src/lib/%s:%s_scss_partials" % l for l in ALL_LIBS]`

Reply:
Added a `.bzl` file at the top-level with a constant for this. I didn't think it was possible to import a constant from another `BUILD` file. Am I missing something?